BlockchainSdk/BlockchainService/Aptos/AptosNetworkService.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  AptosNetworkService.swift
+//  BlockchainSdk
+//
+//  Created by skibinalexander on 29.01.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import Combine
+import SwiftyJSON
+
+class AptosNetworkService: MultiNetworkProvider {
+    // MARK: - Protperties
+    
+    let providers: [AptosNetworkProvider]
+    let blockchainDecimalValue: Decimal
+    
+    var currentProviderIndex: Int = 0
+    
+    // MARK: - Init
+    
+    init(providers: [AptosNetworkProvider], blockchainDecimalValue: Decimal) {
+        self.providers = providers
+        self.blockchainDecimalValue = blockchainDecimalValue
+    }
+    
+    // MARK: - Implementation
+    
+    func getAccount(address: String) -> AnyPublisher<AptosAccountInfo, Error> {
+        providerPublisher { provider in
+            provider
+                .getAccountResources(address: address)
+                .withWeakCaptureOf(self)
+                .tryMap { service, response in
```

Comment:
`withWeakCaptureOf` лишнее как будто
```suggestion
                .tryMap { response in
```

Reply:
> `withWeakCaptureOf` лишнее как будто

Нужно для service.blockchainDecimalValue

Reply:
В остальных местах поубирал, спс